.github/workflows/integration-tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           python-version: '3.13' 
       - uses: actions/setup-java@v4
         with:
-          distribution: 'oracle'
+          distribution: 'temurin'
```

Comment:
why?

Reply:
oracle doesn't support java8 (at least in GitHub actions)

Reply:
Is Java 11 too new for the tests to work?

Reply:
Yes.
https://github.com/scylladb/scylla-ccm/blob/c7854b053e14aeabfeaaaaf11802ca8938ce2aa5/ccmlib/node.py#L196